aliyun/src/main/java/org/apache/iceberg/aliyun/oss/OSSInputStream.java

```diff
@@ -44,10 +48,16 @@
   private long next = 0;
   private boolean closed = false;
 
-  public OSSInputStream(OSS client, OSSURI uri) {
+  private final Counter<Long> readBytes;
+  private final Counter<Integer> readOperations;
+
+  public OSSInputStream(OSS client, OSSURI uri, MetricsContext metrics) {
```

Comment:
Since the old constructor was public, should we keep it and pass `nullMetrics`?

Reply:
Thanks, for the feedback. I was retained the old constructor.

Reply:
Thanks, for the feedback. I have retained the old constructor.

Reply:
Can you make this one package-private? I don't think we need to expose it.